app/backend/src/couchers/models.py

```diff
@@ -896,12 +894,13 @@ class Node(Base):
 
     contained_users = relationship(
         "User",
-        lazy="dynamic",
         primaryjoin="func.ST_Contains(foreign(Node.geom), User.geom).as_comparison(1, 2)",
         viewonly=True,
         uselist=True,
     )
 
+    user_ids = association_proxy("contained_users", "id")
```

Comment:
Not 100% sure this works as intended, as it's empty in the only test where it's used

Reply:
Call this `contained_user_ids`

Reply:
Done. Still planning to write a test for it